independent-publisher-connectors/ShipStationIP/apiProperties.json

```diff
@@ -0,0 +1,34 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "Your ShipStation API Key.",
+          "constraints": {
+            "clearText": true,

```

Comment:
@SmolSoftBoi please add tabIndex properties in constraints section for each connection parameter. Here it will be `"tabIndex": 2`, and for the password property it will be `"tabIndex": 3`. You can refer to the examples provided in the opened issue.